### PR TITLE
[ADD] 9.0 purchase_request_qweb_operating_unit

### PR DESCRIPTION
**Pull request by serpentcs-dev1**
##  **Purchase Request Qweb Report With Operating Unit**

* This module replaces the webkit report by a Qweb report and adds the Operating Unit information to it.

### CODE REVIEW

**Comment by serpentcs-dev1** [comment]
@aheficent Could you please review?

File: purchase_request_qweb_operating_unit/views/report_purchase_request_qweb.xml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+<openerp>
+    <data>
+
+    <template id="external_layout_header_inherit" inherit_id="report.external_layout_header">
+        <xpath expr="//div[@class='header']" position="replace">
+            <t t-foreach="docs" t-as="o">
+                <div class="header">
+                    <div class="row">
+                            <img t-if="company.logo" class="col-xs-2" t-att-src="'data:image/png;base64,%s' % company.logo" style="max-height: 45px; margin-bottom: 4px;"/>
```
Comment (AaronHForgeFlow) [inline purchase_request_qweb_operating_unit/views/report_purchase_request_qweb.xml]: In my opinion this should be the image of partner related to the operating unit

**Comment by AaronHForgeFlow** [comment]
@darshan-serpent purchase_request is now merged in OCA, can you rebase this branch?

**Comment by serpentcs-dev1** [comment]
Rebased the branch, all checks green!!

**Pull request closed without merging**